code/world.dm

```diff
@@ -18,7 +18,7 @@ var/global/datum/global_init/init = new ()
 	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
 
 
-#define RECOMMENDED_VERSION 508
+#define RECOMMENDED_VERSION 510 //Goddammit, update this once in a while!

```

Comment:
I've put worse 'shitpost' comments into the code, really dunno why it's an issue, and it also even tells people to updoot it